### DISCUSSION TITLE
Reset energy

### DISCUSSION THREAD

**Discussion by xoferz** [jmcollin78/versatile_thermostat#229] [Q&A] [answered]
How can I reset the energy used by a thermostat? Since my language uses comma as a decimal separator (not thousands), I seem to have managed to use a small town's energy bill to heat my kitchen. I have fixed the device power value, but cannot find a way to clean the history.

**Comment by jmcollin78** [comment, marked as answer]
You will have to delete and then recreate the VTherm.
